### ISSUE TITLE
Add ability to create python wheels

### ISSUE THREAD

**Issue by rmcgibbo** [pandegroup/openmm#1305]
It might be nice to have the capbility to package OpenMM using python [wheels](https://www.python.org/dev/peps/pep-0427/), which we could upload to PyPI.

This would allow people to install pre-compiled OpenMM with `pip install openmm` from PyPI -- pretty much the same workflow as when working with conda -- except that pip is included by default with all Pythons starting from 2.7.9 / 3.4.

Adding a cmake target for `PythonBdistWheel` would be a big step.

**Comment by rmcgibbo** [comment]
One thing is that with this, you want to let wheel copy the `libOpenMMXYZ.{so,dylib,dll}` into the `lib/pythonX.Y/site-packages/simtk/...` directory, which changes how the rpath stuff works.

**Comment by ncoish** [comment]
This issue is a few years old, but it would be very helpful for my team if we could make this happen to avoid using conda. Just wanted to voice my support and offer to work on this if someone can give me some guidance for the mentioned "rpath stuff" that would need to change.

**Comment by mattwthompson** [comment]
This appears to be done? A good number of platforms are listed: https://pypi.org/project/OpenMM/8.1.1/#files

https://github.com/openmm/openmm-wheels

**Comment by peastman** [comment]
No, it's still in progress.  We have first drafts of the packages for OpenMM itself, but they're still in testing.  And I'm still trying to get packages building for the rest of the ecosystem.

**Comment by peastman** [comment]
We *finally* got the PDBFixer name, and pip packages for PDBFixer and OpenMM Setup are now up.  Which means I can finally close this issue.

This is the second feature request from 2015 I've closed today.  We're on a roll!
